test/functional/tests/initialize/test_initialize_load.py

```diff
@@ -0,0 +1,387 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm, casadm_parser
+from api.cas.cache_config import *
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import *
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.parametrize("core_number", [1, 4])
+@pytest.mark.parametrize("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrize("cache_mode", CacheMode)
+@pytest.mark.parametrize("cache_line_size", CacheLineSize)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_load_x_to_one_without_params(cache_mode, cleaning_policy, cache_line_size, core_number):
+    f"""
+        title: Test for loading CAS with 1 cache and {core_number} cores without extra params.
```

Comment:
not sure if f-string will be properly parsed as documentation string - I'm not saying it won't, I honestly don't know

Reply:
I changed it to normal multi-line string with '1 or 4' phrase instead of variable name.